pkg/util/server/middleware.go

```diff
@@ -22,3 +22,12 @@ func NewPrepopulateMiddleware() middleware.Interface {
 		})
 	})
 }
+
+func ResponseJSONMiddleware() middleware.Interface {
+	return middleware.Func(func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+			w.Header().Add("Content-Type", "application/json; charset=UTF-8")
```

Comment:
I wonder if `Set` would be better here ? So that whatever it was it is replaced. Pretty sure it's no a big deal here, I'll merge it as is tomorrow if you don't have time to make the change.

Reply:
You're right. This should be a replacement, not an `Add`. I'll fix it right now.